tasks/main.yml

```
@@ -49,7 +49,7 @@
   ansible.builtin.file:
     path: /etc/condor/config.d/00-htcondor-9.0.config
     state: absent
-  when: htcondor_version is ansible.builtin.match("10") or htcondor_version is ansible.builtin.match("23")
+  when: htcondor_version is ansible.builtin.match('^(?:10|23|24)(?:\\.|$)')
```

Suggestion: let's go for this so we don't have to come back to the topic in the future?

```suggestion
  when: (htcondor_version.split('.')[0] | int) >= 10
```

---

yes indeed, thanks

---

you can also use ``htcondor_version is version('10.0', '>=')``
See: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/version_test.html

---

> you can also use `htcondor_version is version('10.0', '>=')` See: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/version_test.html

:+1: looks neat and works with mixed int-string values like "24.x", thanks!